buf/registry/module/v1beta1/module.proto

```diff
@@ -65,11 +65,15 @@ message Module {
     (buf.validate.field).string.max_len = 255,
     (buf.validate.field).ignore_empty = true
   ];
-  // The name of the default label of the Module.
+  // The name of the default Label of the Module.
   //
-  // This label may not yet exist. When a Module is created, it has no Commits, and Labels
+  // This Label may not yet exist. When a Module is created, it has no Commits, and Labels
   // must have a Commit, so this Label is not created when a Module is created. Additionally,
   // a User may modify the name of the default Label without this Label yet being created.
+  //
+  // This could also be the name of an archived label. An important note, an archived label
+  // will not be considered a valid ResourceRef for resolving a Module. Archived labels
+  // can only be retrieved using LabelService.GetLabels.
```

Comment:
```suggestion
  // This could also be the name of an archived Label. An important note, an archived Label
  // will not be considered a valid ResourceRef for resolving a Module. Archived Labels
  // can only be retrieved using LabelService.GetLabels.
```